source/guide/3.2/apis-rest-client-python.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
well, we should still provide a link like before to the python client's readthedocs page...